pkg/keyctl/sys_linux.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2015 Jesse Sipprell. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
```

Comment:
Sorry for not noticing this earlier: we need to add `386` and `amd64` to the build constraints as well. The syscall numbers are hard-coded and currently don't support arm, ppc, etc.